presto-main/src/main/resources/webapp/index.html

```diff
@@ -85,13 +85,18 @@
         }
         return errorType;
     },
-    stateClass: function(state)
+    stateClass: function(state, errorType)
     {
         switch (state) {
             case "FINISHED":
                 return "bg-success";
             case "FAILED":
-                return "bg-danger";
+                switch (errorType) {
+                    case "USER_ERROR":
+                        return "bg-warning";
```

Comment:
Seeing cancelled queries separately is important because when the cluster gets backed up or has problems there are often many cancelled queries.